Makefile

```diff
@@ -70,8 +70,8 @@ WIN32_DIR=$(PLATFORM_DIR)/win32
 APPIMAGETOOL=appimagetool-x86_64.AppImage
 APPIMAGETOOL_URL=https://github.com/AppImage/AppImageKit/releases/download/12/appimagetool-x86_64.AppImage
 
-# set to 1 if in Docker
-DOCKER:=$(shell grep -q docker /proc/1/cgroup 2>/dev/null && echo 1)
+# Is fuse support available?
+USE_FUSE ?= $(wildcard /dev/fuse)
```

Comment:
Fwiw the AppImage docs supply the slightly unaesthetic example of `APPIMAGE_EXTRACT_AND_RUN`.

Reply:
Which they decided against documenting in the `--appimage-help` output… *sigh*

Reply:
D'oh, I completely overlooked the implications (or perhaps rather: simplifications) of the *and-run* earlier. They added the `--appimage-extract-and-run` sometime in between 2018 and now, and I failed to realize it was a different option than the `--appimage-extract`.

But that aside. Strangely enough https://gitlab.com/koreader/nightly-builds/-/jobs/6606286131 seems to think it does have fuse, which doesn't make sense to me. :-/

Reply:
Damn. Forget about detection, just use `--appimage-extract-and-run` unconditionally then, since the performance impact is minimal.